src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -87,18 +88,22 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<StatisticsDbContext>(options =>
                 options
                     .UseSqlServer(Configuration.GetConnectionString("StatisticsDb"),
-                        builder =>
-                        {
-                            builder.MigrationsAssembly("GovUk.Education.ExploreEducationStatistics.Data.Model");
-                            builder.AddBulkOperationSupport();
-                        })
+                        providerOptions =>
+                            providerOptions
+                                .MigrationsAssembly("GovUk.Education.ExploreEducationStatistics.Data.Model")
+                                .AddBulkOperationSupport()
```

Comment:
I think `.EnableCustomRetryOnFailure()` is missing here?

Reply:
Don't know how I missed this one, but great spot!  Updated.